weblab/templates/entities/entity_runexperiments.html

```diff
@@ -0,0 +1,80 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+{% load entities %}
+
+{% block title %}Experiment: run specific combinations - {% endblock title %}
+
+{% block body_id %}runexperiment{% endblock %}
+
+{% block content %}
+  <h1 id="entityname">Run experiments using <span>{{ entity.name }}</span></h1>
+    <!-- SK TO DO: make these work -->
+     <p>
+         <input class="overrideCheckBox" type="checkbox" name="rerun_expts"/>
+          Create new versions of existing experiments (if unchecked, existing combinations will be skipped)
+     </p>
+  <div id="runexperiment">
+
+
+    <!-- SK TO DO: make these work -->
+     <button id="checkallbutton">check all</button>
+    <button id="checklatestbutton">check latest</button>
+    <button id="uncheckallbutton">uncheck all</button>
+   <p>
+        <form action="" method="post">
```

Comment:
Possibly you need this to go just under the heading, so all the `input` fields fall within it?